### ISSUE TITLE
FarmOS Core: update to version 3.0

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#606]
Agreed that we would create a Platform.sh site for Aislinn to test before pushing to Rothamsted instances

**Comment by paul121** [comment]
See #592

**Comment by paul121** [comment]
Ah! We will also need to coordinate OAuth API changes, specifically setting up a new Consumer + configure OAuth scopes for needed roles.

**Comment by aislinnpearson** [comment]
Paul to update for Friday 09 Feb, and Aislinn to test on Monday 12th and Thursday 13th, so hopefully can release the week of the 13th.

**Comment by paul121** [comment]
I've created a new `rothamsted` consumer and added scopes for the `rothamsted_data_admin` and `rothamsted_farm_manager` roles. These are the only roles that will be able to use the API, but I think that is OK for now.

This also required making some changes to how our `rothamsted_experiment` plan is rendered in the Form and View displays. Basically, we already had tabs configured for this entity, but we now have a new way of doing this in farmOS core. I've tried to make things appear nearly the same as possible but there a few small changes to the field groups, mostly the "status" group and status fields. TLDR; @aislinnpearson take a look at adding/editing Plans and let me know if there are any issues or if you have questions!
